pages/workshop/Siphon/Siphon Overview.ipynb

```diff
@@ -187,20 +207,24 @@
    "metadata": {},
    "source": [
     "<a name=\"dataaccess\"></a>\n",
-    "## 3. Accessing data"
+    "## 3. Use Siphon to Perform Remote Data Access\n",
+    "\n",
+    "Accessing catalogs is only part of the story; Siphon is much more useful if you're trying to access/download datasets.\n",
+    "\n",
+    "For instance, using our data that we just retrieved:"
    ]
   },
   {
-   "cell_type": "markdown",
-   "metadata": {
-    "slideshow": {
-     "slide_type": "subslide"
-    }
-   },
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
    "source": [
-    "Accessing catalogs is only part of the story; Siphon is much more useful if you're trying to access/download datasets.\n",
-    "\n",
-    "For instance, using our data that we just retrieved:"
+    "# Solution from above in case you had trouble\n",
+    "date = datetime.utcnow() - timedelta(days=1)\n",
+    "cat = TDSCatalog(f'http://thredds.ucar.edu/thredds/catalog/satellite/SFC-T/SUPER-NATIONAL_1km/{date:%Y%m%d}/catalog.xml')\n",
```

Comment:
While we're here, is there a less esoteric dataset we could grab? Given that the GINI products are on their way out...

Reply:
Soooo.. I tried that thinking I'll get a GOES product- like sea surface temps. Filtering on a time range appears to not work with the new file name convention?

Reply:
It should be possible to send the appropriate regex and date format string, but that's probably not a great thing to use in a light intro. Or maybe it is?

Reply:
Well, I'd say regex != light intro, but regex == reality. Maybe we'll give it a shot?

Reply:
Sure.

Reply:
Strike 2 - no julian day support the way that setup currently works.

Reply:
*sigh* That's because that support is sitting in master. 😠 Ugh.  Level2?